app/jobs/document_proofing_job.rb

```diff
@@ -5,15 +5,13 @@ class DocumentProofingJob < ApplicationJob
 
   discard_on JobHelpers::StaleJobHelper::StaleJobError
 
-  # rubocop:disable Lint/UnusedMethodArgument
   def perform(
     result_id:,
     encrypted_arguments:,
     trace_id:,
     image_metadata:,
     analytics_data:,
-    flow_path:,
-    liveness_checking_enabled: nil
```

Comment:
`perform` is the job arguments, `liveness_checking_enabled` is the one we need to keep I think? re: https://github.com/18F/identity-idp/pull/7240/files#r1007213954

Reply:
Yes, this argument needs to say